src/components/Availability.js

```diff
@@ -9,22 +9,25 @@ export default function Availability({entry, onlyShowAvailable}) {
     ) {
         return (
             <div>
-                {entry.totalAvailability}
-                {entry.totalAvailability === 1 ? "slot" : "slots"}
+                {`${entry.totalAvailability} ${
+                    entry.totalAvailability === 1 ? "slot" : "slots"
+                }`}
```

Comment:
This was breaking a test because it was inserting two different text nodes. I am guessing these were supposed to be one line with a space between them, eg. "45 slots"? If not lemme know and I'll revert